build-system/tasks/prepend-global/index.js

```diff
@@ -284,34 +276,36 @@ async function prependGlobal() {
       'build-system/global-configs/prod-config.json'
     );
   }
-  await removeConfig(target);
-  return applyConfig(
-    config,
-    target,
-    filename,
-    argv.local_dev,
-    argv.local_branch,
-    argv.branch,
-    argv.fortesting,
-    argv.derandomize
-  );
+  await Promise.all([...targets.map(removeConfig)]);
+  return Promise.all([
```

Comment:
nit: Since this is the last line in an `async` function, it can just be `await Promise.all()`, right?

Reply:
From what I understand `async function func() { return somePromise; }` translates to `function func() { return new Promise((resolve, reject) => { resolve(somePromise); }); } so adding await would make this `function func() { return new Promise((resolve, reject) => { resolve(await somePromise); }); } which doesn't do anything

Reply:
I think if you add a `return` to line 282 and make it `return applyConfig(...)`, you can simply `await Promise.all(...)` on line 282. (If I'm wrong, feel free to ignore.)

Reply:
Line 282 is part of the lambda, it already returns :)

Reply:
Gotcha. Maybe there's something else at play that's evading me right now. In any case, this nit is unimportant and needn't hold back this PR.